rubix-spi/src/main/java/com/qubole/rubix/spi/CacheConfig.java

```diff
@@ -86,6 +86,7 @@
   private static final String KEY_MAX_CACHE_SIZE = "rubix.cache.max.size";
   private static final String KEY_CACHE_FILE_SPLIT_SIZE = "rubix.cache.filesplit.size";
   private static final String KEY_CLUSTER_NODE_REFRESH_TIME = "rubix.cluster.node.refresh.time";
+  private static final String KEY_ENABLE_PATH_ENCRYPTION = "rubix.cache.path.encryption";
```

Comment:
Add `.enabled` to make it clearer that this is a key for a Boolean config option

Reply:
Done